<launch>
  <node pkg="team1" type="obstacle_avoidance_client.py" name="obstacle_avoidance_client" output="screen">
  </node>
  <node pkg="team1" type="obstacle_avoidance_explore_server.py" name="explore_action_server" output="screen">
  </node>
</launch>